Presentation.Web/app/models/it-system-usage/it-system-usage-life-cycle-status.ts

```diff
@@ -0,0 +1,83 @@
+﻿module Kitos.Models.ItSystemUsage {
+    export enum LifeCycleStatusType {
+        Undecided = 0,
+        NotInUse = 1,
+        PhasingIn = 2,
+        Operational = 3,
+        PhasingOut = 4
+    }
+    
+    export class LifeCycleStatus {
+        static readonly Undecided = {
+            Text: Constants.Select2.EmptyField,
+            EnumStringValue: "Undecided"
+        }
+
+        static readonly PhasingIn = {
+            Text: "Under indfasning",
```

Comment:
might as well put the enum value in there as well and then you can actually do the mapping more easily